app/views/rooms/index.html

```diff
@@ -0,0 +1,23 @@
+<h1>List Rooms</h1>
+<div class="row">
+  <div class="col-md-12">
+    <div ng-repeat="room in room.listRooms" class="container">
```

Comment:
Should add `track by` room id

ref https://docs.angularjs.org/api/ng/directive/ngRepeat